package.json

```diff
@@ -92,65 +93,72 @@
     "redux-thunk": "^2.2.0",
     "reselect": "2.5.4",
     "serialize-javascript": "^1.4.0",
-    "slug": "^0.9.1",
     "source-map-support": "^0.5.8",
     "validator": "^9.2.0",
     "victory": "^0.25.7",
     "websocket.js": "^0.1.7"
   },
   "devDependencies": {
+    "@babel/core": "^7.3.3",
+    "@babel/plugin-proposal-class-properties": "^7.3.3",
+    "@babel/plugin-syntax-dynamic-import": "^7.2.0",
+    "@babel/preset-env": "^7.3.1",
+    "@babel/preset-flow": "^7.0.0",
+    "@babel/preset-react": "^7.0.0",
+    "@babel/preset-stage-2": "^7.0.0",
     "assets-webpack-plugin": "^3.5.1",
-    "babel-core": "^6.23.1",
-    "babel-eslint": "^8.0.1",
-    "babel-jest": "^21.0.2",
-    "babel-loader": "^7.1.1",
+    "babel-eslint": "^10.0.1",
+    "babel-jest": "^24.1.0",
+    "babel-loader": "^8.0.5",
     "babel-plugin-lodash": "^3.2.11",
     "babel-plugin-transform-import-to-require": "^1.0.0",
     "babel-polyfill": "^6.23.0",
     "babel-preset-latest": "^6.22.0",
-    "babel-preset-react": "^6.23.0",
+    "babel-preset-react": "^6.24.1",
     "babel-preset-stage-2": "^6.22.0",
     "concurrently": "^3.5.0",
-    "css-loader": "^0.28.0",
     "enzyme": "^3.9.0",
-    "enzyme-adapter-react-16": "^1.9.1",
-    "eslint": "^4.9.0",
-    "eslint-config-prettier": "^2.7.0",
-    "eslint-config-react-app": "^2.0.1",
-    "eslint-import-resolver-webpack": "^0.8.3",
-    "eslint-plugin-flowtype": "^2.39.1",
+    "enzyme-adapter-react-16": "^1.0.2",
+    "eslint": "^5.14.1",
+    "eslint-config-prettier": "^4.1.0",
+    "eslint-config-react-app": "^3.0.7",
+    "eslint-import-resolver-webpack": "^0.11.0",
+    "eslint-plugin-flowtype": "^3.4.2",
     "eslint-plugin-import": "^2.8.0",
-    "eslint-plugin-jsx-a11y": "5.1.1",
-    "eslint-plugin-react": "^7.4.0",
+    "eslint-plugin-jsx-a11y": "^6.2.1",
```

Comment:
Bump to `6.2.0` to so we can get this aswell https://github.com/webkom/lego-webapp/pull/1415